src/tests/src/libs/antares/test_utils.cpp

```diff
@@ -18,6 +18,13 @@ T beautify(const T& in) {
 constexpr auto beautifyStd = beautify<std::string>;
 constexpr auto beautifyYuni = beautify<Yuni::String>;
 
+std::string transformNameToId(const AnyString& name)
```

Comment:
Is it a problem to test **Antares::TransformNameIntoID** directly without the help of **transformNameToId** ?

Reply:
It's just less readable: this small function allows to have one-liners in the actual test code.

Reply:
Then, as this new function's calls are more readable, and now we have tests for it, wouldn't be easy to (to be finished...)

Reply:
As discussed : I agree that the function should be changed in actual code, but it's out of the scope of this PR.
We should make a dedicated PR for this change, which will impact many places in the code base!